### PR TITLE
LPS-135504 Language Key Conflicts

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-135504

### CODE REVIEW

File: modules/apps/dynamic-data-lists/dynamic-data-lists-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -34,6 +33,7 @@ edit-display-template=Edit Display Template
 edit-form-template=Edit Form Template
 form-template=Form Template
 form-view=Form View
+is-default-data-definition-language-changeable=If checked, the default language of data definitions will be changeable. The default language of data definitions is inherited from the site settings.
```
Comment (Preston-Crary) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-lang/src/main/resources/content/Language.properties:36]: Key should be `if-checked,-the-default-language-of-data-definitions-will-be-changeable`

File: modules/apps/segments/segments-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -118,6 +117,7 @@ javax.portlet.title.com_liferay_segments_web_internal_portlet_SegmentsPortlet=Se
 key-is-already-used=Key is already used.
 less-than=Less Than
 less-than-or-equals=Less Than or Equals
+liferay-dxp-instance-has-to-be-connected-with-analytics-cloud-to-perform-an-ab-test=In order to perform an A/B test, your Liferay DXP instance has to be connected with Liferay Analytics Cloud.
```
Comment (Preston-Crary) [inline modules/apps/segments/segments-lang/src/main/resources/content/Language.properties:121]: Key should be `in-order-to-perform-an-ab-test,-your-liferay-dxp-instance-has-to-be-connected`

File: modules/apps/users-admin/users-admin-web/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -66,7 +66,7 @@ security-questions=Security Questions
 select-the-delivery-options-for-alerts-and-announcements=Select the delivery options for alerts and announcements.
 specify-custom-security-questions-for-the-users-of-this-organization=Specify custom security questions for the users of this organization. Enter one question per line.
 the-organization-name-is-already-taken=The organization name is already taken.
-the-password-you-entered-for-the-current-password-does-not-match-your-current-password=The password you entered for the current password does not match your current password. Please try again.
+the-password-you-entered-for-the-current-password-does-not-match-your-current-password-please-try-again=The password you entered for the current password does not match your current password. Please try again.
```
Comment (Preston-Crary) [inline modules/apps/users-admin/users-admin-web/src/main/resources/content/Language.properties:69]: Key should be `the-password-you-entered-for-the-current-password-does-not-match-your-current-password.-please-try-again`

**Comment by Preston-Crary** [comment]
Please see my comments and double check the other keys include commas and periods in the keys. Only rephrase/shorten if it's really long. We have some keys that are 125 characters, I think anything shorter than that doesn't need rephrasing. 3f5a532 has 150 characters which is why I rephrased it. It's better to just shorten it to a sentence ending if it's a paragraph.

**Comment by ctang3** [comment]
@Preston-Crary Applied requested changes, thanks!

File: modules/dxp/apps/sharepoint-soap/app.bnd-localization/bundle_zh_TW.properties [hunk 4]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+liferay-releng-app-description=Liferay SOAP Connector for SharePoint 應用程式使用戶可以從Liferay 文件和媒體庫中與 SharePoint 庫中的文件進行交互。此應用程式在 "文件和媒體庫" 中提供了一種 SharePoint 庫類型，用於安裝 SharePoint 庫。在這樣的存儲庫中，用戶可以查看、添加、修改和移動文件和資料夾。該應用程式將在 SharePoint 庫中執行的操作散播到 Liferay 文件和媒體庫中的 SharePoint 存儲庫，反之亦然。<br /> <br />以下是該應用程式的一些功能：<br /> <br /> *讀取/寫入文件和資料夾<br /> *文件簽入、簽出和撤消簽出<br /> *下載文件<br /> *在存儲庫中移動文件夾和檔案<br /> *獲取修訂歷史記錄<br /> *恢復為修訂版本<br /> <br />該應用程式的當前版本可與 SharePoint 2013 一起使用。<br /> <br />立即將Liferay Digital Experience Platform 與 SharePoint 庫連接起來！ <br /> <br />重要更改：該應用程序的名稱最近已更改。請造訪https://help.liferay.com/hc/en-us/articles/360018233992 以了解有關安裝過程的更多資訊。
```
Comment (Preston-Crary) [inline modules/dxp/apps/sharepoint-soap/app.bnd-localization/bundle_zh_TW.properties:1]: Please fixup this commit to not touch this file, thanks!

Comment (ctang3) [reply modules/dxp/apps/sharepoint-soap/app.bnd-localization/bundle_zh_TW.properties]: Reverted that change

**Comment by Preston-Crary** [comment]
ci:forward

**Comment by ctang3** [comment]
ci:forward

**Comment by ctang3** [comment]
ci:reevaluate:1267401_234

**Comment by ctang3** [comment]
ci:test:relevant

**Comment by ctang3** [comment]
@Preston-Crary Fixed the causes of most of the failing tests, but I think the `SharepointConnectionTest` failure is not due to this PR's changes. The [build log](https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/9198/testReport/com.liferay.sharepoint.soap.repository.connector/SharepointConnectionTest/classMethod/) shows that `SharepointConnectionTestUtil.getSharepointVMHostname()` called by `SharepointConnectionTest.java:60` is failing to get the Sharepoint VM hostname and `SharepointConnectionTestUtil` doesn't reference any language keys.

**Comment by Preston-Crary** [comment]
f129eda is the wrong fix, I broke the xml elements in eb53f49 by changing a matching string I wasn't supposed to. I'll fix it and resend later today. Otherwise we would need to create an upgrade to swap the xml elements already in the DB.

**Comment by Preston-Crary** [comment]
Please see https://github.com/Preston-Crary/liferay-portal/pull/1454

**Pull request closed without merging**